### ISSUE TITLE
[CAIA intake] Create new Unified Medical Supply Reordering Page

### ISSUE THREAD

**Issue by JasmineYohannan** [department-of-veterans-affairs/va.gov-team#79661]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Supply Reordering 
- **OCTO product owner:** `Patrick Bateman/Jasmine Yohannan 
- **Product name:** `Insert product name`
- **Product manager:**  `Jasmine Yohannan
- **Slack channel:** va-cto-supply-reordering 
- **Dedicated content writer on your team (if you have one):** `N/A
- **Dedicated a11y specialist on your team (if you have one):**  N/A
 
 
## About your initiative
Supply Reordering digitizes a paper form so Veterans can order hearing aid accessories, batteries and CPAP supplies on VA.gov. 

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [X] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [X ] Something else:  merging existing content pages together

### What's the nature of your initiative and desired outcomes?

.We want to combine the hearing aid supplies and cpap supplies static landing pages into a single supply page in hopes of simplifying navigation.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [X] I’m not sure

### Where are you at in your timeline?
Would like to have this done in May if possible. 
### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [X] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact <name>](https://va.gov) `placeholder link`

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83167
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83445
```

```[tasklist]
### CAIA IA tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/80187
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499
```

Health
```[tasklist]
### Related CC intakes
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/86264
```

**Comment by mnorthuis** [comment]
@strelichl Please connect with @DanielleThierryUSDSVA on this one - she has an available resource that can assist that will allow this to move forward, and IA resources are available as well. Thanks!

**Comment by kristinoletmuskat** [comment]
Hey @JasmineYohannan and @laurwill and FYI @sterkenburgsara I put together a draft IA plan for the unified supply reordering page -- the biggest open Q for IA is whether we have enough content on the 3 separate devices (hearing aid, cpap, prosthetic socks) to warrant static pages in the hub about them, or if we want to create R+S pages with that content and keep the pages in the hub to a minimum. 

Here's the draft IA spec work: https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1712349672737/42d77ae33ad84086e68e98d4a06b8daa33303fc8?sender=ua67f17f1c416a96ea04d2476

**Comment by mnorthuis** [comment]
We should not do option 1 - we should be working towards a consolidated single page rather than adding a level and extra click to the flow.  Much of the content can move off to R&S, and content regarding how to submit orders for each of these items should remain on a consolidated page.  We should reference SEMrush/SEO data to determine how to craft an H1 and URL, as well as meta description for this new consolidated page so we maintain the values we are getting from the individual pages.

**Comment by kristinoletmuskat** [comment]
Here is the IA spec in github! https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/unified-medical-supplies-ordering.md

**Comment by kristinoletmuskat** [comment]
health (test)

**Comment by laurwill** [comment]
Hi @mnorthuis @kristinoletmuskat , 

We also have a ticket with Public Websites to replace the action links on the hearing aids and CPAP pages with sign-in widgets (needed before we can migrate those forms to the health apartment). 

To avoid unnecessary work, can we pause on adding the widgets to those 2 existing pages, since we're planning to create a unified page? We'll have to do some content work on the page hierarchy etc to fit the widgets in, and it seems better to avoid that if we're about to blow up those pages entirely.

**Comment by mnorthuis** [comment]
@laurwill I think it depends on the timing of everything.  If we can get the unified page complete prior to the intended move of the existing order form into my health, then I'd say yes, we can pause on that widget work.

**Comment by kristinoletmuskat** [comment]
Priority update: this is in our content backlog, but we're waiting to see if we can get a content person from VHA to step in and work on this. Danielle is looking into that.

**Comment by laurwill** [comment]
@kristinoletmuskat the IA spec here shows a link to ebenefits for prosthetic socks. But ebenefits is sunsetting, right? The live page doesn't link there, just provides phone or mail options.

**Comment by JasmineYohannan** [comment]
Hi @laurwill confirming that the only SME review would be Patrick Booth from DLC. They may want to review it with others from DLC but that is the main POC.

**Comment by laurwill** [comment]
Updates here! 
- Patrick Booth answered our questions and approved the page updates
- I let him know we would be publishing the page and redirecting the 3 existing pages
- We published the page and the redirects are live

@kristinoletmuskat when you're online today, the last step is to change the left nav and megamenu to point to the new page. I can do the left nav, but do you prefer to do the megamenu change yourself?

**Comment by kristinoletmuskat** [comment]
Hey ya'll, we updated the left nav, set up best bet (va.gov search), and updated IA documentation. 

Last step from our redirect process is to update the drupal nodes and deprecate the old pages, but neither of these steps holds up our next phase (adding sign-in widget and moving form to my-health)

@laurwill @JasmineYohannan

**Comment by kristinoletmuskat** [comment]
The work for this ticket is complete, so closing!
